src/docs/product/error-monitoring/states-triage/index.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Issue States and Triage
+sidebar_order: 30
+description: "Learn more about issue states and how to triage issues."
+---
+
+An issue can be in one of three states:
+
+1. Unresolved: The default state for all new issues.
+2. Ignored: The issue is ignored permanently or until a condition is met to move it to a state of Unignored. [Issue alerts](/product/alerts-notifications/issue-alerts/) are suppressed for ignored issues.
+3. Resolved: The issue is resolved because it was fixed. You can also mark an issue as Resolved if you don't expect it to happen again.
+
+The "For Review" tab, or Review List, of the Issues page displays issues when any of the following happens:
+
+- A new issue is created; it's labeled "New Issue"
+- A resolved issue re-occurs; it's labeled "Regression"
+- An ignored issue meets its ignore conditions; it's labeled "Unignored"
+
+For-review issues are a subset of all unresolved issues.
+
+## Issue Triage
+
+Triaging is the act of deciding what action you want to take on an issue. It typically involves one or more of these actions.
```

Comment:
> To help you organize issues for you and your team you can assign, resolve, review, or ignore an issue. These actions are referred to as "triaging."

Reply:
I think we should move this description to the top. Explain roughly what triaging is, then explain the states and actions.

Reply:
Can do. Suggest the following edit:

```
To help organize issues for you and your team, you can take actions such as assigning, resolving, reviewing, or ignoring issues. These actions are referred to as _triaging_.
```